c/ssh-simplified/s3_srvr_1_true-unreach-call.cil.c

```diff
@@ -5,6 +5,9 @@ extern int __VERIFIER_nondet_int(void);
 extern long __VERIFIER_nondet_long(void);
 extern void *__VERIFIER_nondet_pointer(void);
 extern int __VERIFIER_nondet_int();
+
+#include <stdio.h>
```

Comment:
This adds an include to a file that is supposed to have none. If it misses just a few declarations, I would suggest to inline them manually as you did in other files.